### ISSUE TITLE
README: which ansible-playbook command should I use for Ubuntu VM ?

### ISSUE THREAD

**Issue by niconoe** [AtlasOfLivingAustralia/ala-install#44]
The first suggestion in the README file is:

    $ ansible-playbook -i inventories/vagrant ala-demo.yml --private-key ~/.vagrant.d/insecure_private_key -u root

(SSH then refuses the private_key)

I had better results by altering the command line to log in with vagrant user and to use sudo to gain root privileges:

    $ ansible-playbook --sudo --ask-sudo-pass -i inventories/vagrant ala-demo.yml --private-key ~/.vagrant.d/insecure_private_key -u vagrant

That seems to work well, but I finally encounter an error at the "create collectory DB" task: 

    failed: [vagrant1] => {"failed": true}
    msg: unable to connect, check login_user and login_password are correct, or alternatively check        ~/.my.cnf contains credentials

I don't know if this error or related or not, nor if my proposal ($ ansible-playbook --sudo --ask-sudo-pass -i inventories/vagrant ala-demo.yml --private-key ~/.vagrant.d/insecure_private_key -u vagrant) is correct. Could you clarify, and update the documentation if necessary ?

Best, n.

**Comment by m-r-c** [comment]
Hi,

Using "-u vagrant" is correct. I have updated the doco accordingly.

The issue with the collectory db task has also been fixed - the mysql role was missing a task to create the .my.cnf file on the server for the MySQL credentials. You will need to force the MySQL installation tasks to run again. These easiest way to do that is to just destroy and recreate your vagrant VM, then run the playbook again.
